lib/registry.ts

```diff
@@ -101,7 +101,6 @@ export class Registry {
     this.runStepDefininition = this.runStepDefininition.bind(this);
     this.defineParameterType = this.defineParameterType.bind(this);
     this.defineBefore = this.defineBefore.bind(this);
-    this.defineAfter = this.defineAfter.bind(this);
```

Comment:
This is probably not right.

Reply:
Deleted this by mistake, reverting it back